### ISSUE TITLE
Unable to retrieve particular documents

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-omg#12]
<img width="1179" alt="image" src="https://github.com/relaton/relaton-omg/assets/11865/fe6f69a3-22d0-4e7d-aa38-8306597d3000">

Here you can see that there are two separate specifications:
* OMG UML/2.1.1/Superstructure/PDF
* OMG UML/2.1.1/Infrastructure/PDF

We should be able to fetch them. Right now, we can only fetch "OMG UML/2.1.1" but is not specific enough.

**Comment by andrew2net** [comment]
@ronaldtse fixed in v 1.18.1
```sh
$ relaton fetch 'OMG UML 2.1.1 Superstructure'
relaton-omg] (OMG UML 2.1.1 Superstructure) Fetching from www.omg.org ...
[relaton-omg] (OMG UML 2.1.1 Superstructure) Found: `UML 2.1.1 Superstructure`
<bibdata schema-version="v1.2.8">
  <fetched>2024-06-27</fetched>
  <title type="main" format="text/plain" language="en" script="Latn">Unified Modeling Language: Superstructure</title>
  <uri type="src">https://www.omg.org/spec/UML/2.1.1/Superstructure/PDF</uri>
  <docidentifier type="OMG" primary="true">UML 2.1.1 Superstructure</docidentifier>
  <date type="published">
    <on>2007-07</on>
  </date>
  ...
```

**Comment by ronaldtse** [comment]
Thanks @andrew2net !
